packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -573,3 +569,9 @@ export const SDP = {
   CARRIAGE_RETURN: '\r\n',
   BAD_MEDIA_PORTS: [9, 0]
 };
+
+export const HTTP_VERBS = {
```

Comment:
👍 This is a good example of the grouping I spoke of in other comments, eliminating the need for `_PUT_`, etc.